tests/unit/database_cache_backend_test.py

```diff
@@ -0,0 +1,35 @@
+import time
+
+import pytest
+
+from caching.backends import DatabaseBackend
+
+
+@pytest.mark.asyncio
+async def test_backend(tmpdir):
+    backend = DatabaseBackend(url=f"sqlite:///{tmpdir}/test.db")
+    await backend.setup()
+
+    assert await backend.get("foo") is None
+    await backend.set("foo", "bar", 100)
+    assert await backend.get("foo") == "bar"
+    assert await backend.get_with_ttl("foo") == (100, "bar")
+    time.sleep(3)
```

Comment:
I presume 1s would be fine here too, right? Or does it take 3s for the cache to really show changes?

Reply:
It should - The whole test is a little wonky relying on time.sleep(). If it sleeps a little to long or to short, it would fail...

Reply:
In an ideal world, we could mock time and just move it forward. I think in an old project we had some code on this. But it is fine for now :) Thanks for clarifying.